ethpm_types/abi.py

```diff
@@ -84,6 +84,24 @@ def signature(self) -> str:
         return "fallback()"
 
 
+class ReceiveABI(BaseModel):
+    type: Literal["receive"]
+
+    # No `name` field
+    stateMutability: str = "payable"  # NOTE: Should be either "payable" or "nonpayable"
+
+    @property
+    def is_payable(self) -> bool:
+        return self.stateMutability == "payable"
+
+    @property
+    def signature(self) -> str:
+        """
+        String representing the function signature for display purposes only.
+        """
+        return "receive()"
```

Comment:
`receive()` seems to always require `external` and `payable`. Should we include those modifiers here in the signature then?

Reply:
I don't think these signatures ever have `external` in them, or at least *all* ABI functions are `external` (or at least `public` which is `external + internal` )

`payable` might make sense to include. My intended use case for these was in our console being able to display it with tab-completion eventually.